packages/angular/ssr/node/test/BUILD.bazel

```diff
@@ -7,16 +7,13 @@ ts_project(
     srcs = glob(["**/*_spec.ts"]),
     deps = [
         "//:node_modules/@types/node",
-        "//packages/angular/ssr/node:node_rjs",
+        "//packages/angular/ssr/node",
     ],
 )
 
 # TODO: Clean this up when this repo runs ESM consistently.
```

Comment:
Is this cleaned up now then? Or is this TODO about something else.

Reply:
This is about executing ESM in the repo. Unrelated to the migration.